lib/maths/unittest/CBoostedTreeTest.cc

```diff
@@ -114,33 +101,385 @@ void CBoostedTreeTest::testPiecewiseConstant() {
             LOG_DEBUG(<< "function moments = " << functionMoments);
             LOG_DEBUG(<< "model prediction error moments = " << modelPredictionErrorMoments);
 
-            // Unbiased...
-            CPPUNIT_ASSERT(maths::CBasicStatistics::mean(modelPredictionErrorMoments) < 0.01);
```

Comment:
Are we now asserting something here?

Reply:
This is an artefact of reworking the tests, i.e. this `CPPUNIT_ASSERT` is compared against a helper function which extracts the statistics the tests assert on.